README.md

```diff
@@ -17,94 +25,136 @@
 
 ### Chapter 2: Getting Started with Dash
 
- - Structure of a Dash app: data, layout, callback
- - Installation
+ - Structure of a Dash app
+ - Installation and setup
+ - Create your first Dash app
+ - Live updates and debugging
 
-### Chapter 3: Dash Components and Layout
+### Chapter 3: Dash Components and Layouts
 
- - Core components: dcc.dropdown, dcc.graph 
- - HTML components: html.div
  - Design the app layout
-
-### Chapter 4: Callback I- connecting two components
-
- - Callback Decorator: Input, Output, Component_id, Component_property
- - Callback function
+ - Linking a CSS file
+ - Dash Core components
+   - Markdown
+   - Dropdown
+   - Checkbox
+   - Slider
+   - Graph
+   - etc...
+ - Dash Bootstrap Components:
+   - Container
+   - Accordion
+   - Card
+   - Layout
+   - Row and Col
+
+### Chapter 4: Linking Dash components
+
+ - Introduction to decorators in Python
+ - Callback decorators in Dash
+   - Input
+   - Output
+   - Component_id
+   - Component_property
+ - Setting up a callback
+   - Dropdown and Graph
+   - Slider and Graph
  
-### Chapter 5: Deploying app to the Web
+### Chapter 5: Deploying a Dash app to the Web
 
  - Basic deployment to Heroku
 
 ## Part II: Data Visualization
 
- - What you will learn in this part
+- Summary and overview of this part
+- Learning Intentions
 
-### Chapter 1: Using Data inside App
+### Chapter 6: Working with Data in Dash
 
- - Loading data: creating our own pandas df, from excel to df, connecting to api 
- 
-### Chapter 2: Wrangling data
+- Using an existing pandas dataframe
```

Comment:
what's exactly an existing pandas dataframe? Do you mean take a pandas dataframe that was already created by someone else?

Reply:
Oops, here I meant:

`pd.read_csv('https://....csv')

Reply:
I've resolved this in the doc.